sk/src/lib/components/ReleaseCard.svelte

```diff
@@ -62,11 +66,11 @@
     <Card.Footer>
       <div class='grid grid-cols-2 gap-4 w-full'>
         {#each torrents as torrent}
-          <Button size='sm' variant='outline' class='px-4 {PRIVATE_TRACKERS.includes(torrent.tracker) ? 'hidden pt-button' : ''}' href={torrent.url} target='_blank'>
+          <Button size='sm' variant='outline' class='px-4 {PRIVATE_TRACKERS.includes(torrent.tracker) ? 'pt-button' : ''}' href={!PRIVATE_TRACKERS.includes(torrent.tracker) ? torrent.url : '#'} data-href={PRIVATE_TRACKERS.includes(torrent.tracker) ? torrent.url : ''} target='_blank'>

```

Comment:
data-href doesnt need the condition, just inline it, href itself doesnt need to be "#" it can be an empty string